packages/compass-crud/src/components/bulk-update-dialog.tsx

```diff
@@ -195,6 +195,9 @@ export default function BulkUpdateDialog({
             <ReadonlyFilter
               queryLabel="Filter"
               filterQuery={toJSString(filter) ?? ''}
+              onExportToLanguage={() => {
+                return;
+              }}
```

Comment:
Is this some sort of leftover?

Reply:
No, this is an integration with a refactor @lerouxb worked on. He will need to implement this function, but added an empty one so it at least compiles on CI.

Reply:
Let's add a todo with a ticket number?

Reply:
Done :D